drulenium_visual_testing/post_drulenium_github.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+// An example of usign Pantheon's Quicksilver technology to do 
+// automatic visual regression testing using Drulenium
+
+// Modify default Test definition accordingly.
+/* Example
+$test_definition = array (
+  'staging_environment_domain' => 'http://test-drulenium-hosted.pantheonsite.io/',
+  'dev_environment_domain_suffix' => '-drulenium-hosted.pantheonsite.io/',
+  'notification_email_ids' => array(
+    'drulenium@mailinator.com',
+  ),
+  'test_pages_sitemap' => array(
+    'boston',
+    'new-york',
+    'node',
+  ),
+);
+*/
+$test_definition = array (
+  'staging_environment_domain' => 'http://test-<add-my-site-name>.pantheonsite.io/',
+  'dev_environment_domain_suffix' => '-<add-my-site-name>.pantheonsite.io/',
+  'notification_email_ids' => array(
+    'add-my-email-here',
+  ),
+  'test_pages_sitemap' => array(
+    'add-my-first-page-here',
+    'add-my-second-page-here',
+    'add-my-third-page-here',
+  ),
+);
+/*----------END of REQUIRED Simple Configuration----------*/
+
+// Modify below variables if you are using your personal github account like https://github.com/Drulenium/pantheon-travis
+/*----------START of OPTIONAL Advanced Configuration-----------*/
+$github_username = 'Drulenium';
+$github_repository = 'pantheon-travis';
+$github_accesstoken = 'a06e6d536db8743056e1faae60aa803a0b17b13f';
```

Comment:
@ndubbaka Github access token, email address and other items shouldn't be committed to version control. There can be a `private` directory on each Pantheon environment. It's best to create a `secrets.json` file in that directory and store private items there. See the [Slack integration example `README.md`](https://github.com/pantheon-systems/quicksilver-examples/blob/master/slack_notification/README.md) as an example.